packages/fxa-content-server/app/scripts/models/account.js

```diff
@@ -1644,6 +1644,20 @@ const Account = Backbone.Model.extend(
         )
         .then(this.set.bind(this));
     },
+
+    /**
+     * Verify the OIDC ID Token associated with an account.
+     *
+     * @param {String} idToken - the ID Token
+     * @param {String} clientId - the client ID, used to verify the 'aud' claim
+     * @throws `invalidToken` error if Token is invalid.
+     * @returns {Promise<Object>} resolves with claims in the Token
+     */
+    verifyIdToken(idToken, clientId) {
+      const sessionToken = this.get('sessionToken');
+
+      return this._fxaClient.verifyIdToken(idToken, clientId, sessionToken);
```

Comment:
You got a `sessionToken` on the end of your args here that doesn't feel like it should be necessary (as noted above, this endpoint doesn't do anything secret so you shouldn't need any secrets in order to access it).